src/app/oauth_config.ts

```diff
@@ -0,0 +1,13 @@
+import { environment } from "../environments/environment";
+import { CustomConfig } from "ng2-ui-auth";
+
+
+export const GOOGLE_CLIENT_ID = '493384436662-adfvbtv8otrc25kplnldokq4tlar545j.apps.googleusercontent.com';
```

Comment:
@laertispappas this could be improved IMO, @pkrawat1 what do you think? IMO this should be configurable on the backend in the .env file or something and then can be fetched over a request. Thoughts?

Reply:
Yes, should be, I was looking for some alternative for accessing and loading .env file in angular, ju8st like we have in react , it auto loads .env files in process.

Reply:
Yaa that would be ideal, in any case system environment vars should always work and that would be easy to implement!

Reply:
@ashish173 I am not sure I do know how to make the http call at this point.